### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #1: Create a pull request Template
Create a pull request that covers all the requirements for A perfect Pull request and is not limited to:
- Perfect code style check
- successful build
- No console errors
- No console warnings (sometimes)
- Pull request against branch name
- responsive UI
- Test case being passed

### PR DESCRIPTION
**Pull request by atharmohammad**
### Create Pull_request Template

Fixes  #1

### CODE REVIEW

**Comment by atharmohammad** [comment]
@KeenWarrior I have created pull request template tell me if any changes are required

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+### Description
+
+<!--- Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change. --->
+
+### Related Issue
```
Comment (Abhishek-kumar09) [inline .github/pull_request_template.md:5]: Make it Bold, instead of h3

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+### Description
```
Comment (Abhishek-kumar09) [inline .github/pull_request_template.md:1]: Make it Bold

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### Type of Change:
+
+<!--- **Delete irrelevant options.** --->
+
+- Code
+- User Interface
+- New Feature
+- Documentation
+- Testing
```
Comment (Abhishek-kumar09) [inline .github/pull_request_template.md:20]: How about checkboxes? Is it good to put here?

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Remove all h3 (###) into `**` Bold texts

File: .github/pull_request_template.md (hunk 3 above)
Comment (atharmohammad) [reply .github/pull_request_template.md:20]: Yeah we can do it ...I will  do all changes you asked for

**Comment by atharmohammad** [comment]
@Abhishek-kumar09  I have done the changes tell me if something more you want me to add or change

**Pull request merged**